### PR TITLE
Add Admonitions to Maintain >> Kusama Content

### PR DESCRIPTION
**Pull request by salmad3**
Part of https://github.com/w3f/polkadot-wiki/issues/3133

### CODE REVIEW

File: docs/maintain/kusama/maintain-guides-how-to-nominate-kusama.md [hunk 1]
```diff
@@ -6,8 +6,12 @@ description: Steps on how to nominate on Kusama.
 slug: ../../maintain-guides-how-to-nominate-kusama
 ---
 
-> The following information applies to the Kusama network. If you want to nominate on Polkadot,
-> check out the [Polkadot guide](../maintain-guides-how-to-nominate-polkadot.md) instead.
+:::info The following information applies to the Kusama network
```
Comment (profradha) [inline docs/maintain/kusama/maintain-guides-how-to-nominate-kusama.md]:
```suggestion
:::info 
```

File: docs/maintain/kusama/maintain-guides-how-to-nominate-kusama.md [hunk 2]
```diff
@@ -6,8 +6,12 @@ description: Steps on how to nominate on Kusama.
...
+
+If you want to nominate on Polkadot,
```
Comment (profradha) [inline docs/maintain/kusama/maintain-guides-how-to-nominate-kusama.md]:
```suggestion
The following information applies to the Kusama network. If you want to nominate on Polkadot,
```

File: docs/maintain/kusama/maintain-guides-how-to-nominate-kusama.md [hunk 3]
```diff
@@ -94,8 +102,12 @@ earn compound interest.
 
 ![Payout account selection dropdown with the custom account option highlighted](../../assets/payout/01.png)
 
-> These concepts have been further explained in Polkadot's
-> [UI Walkthrough Video](https://www.youtube.com/watch?v=mNStMPZjiHM&list=PLOyWqupZ-WGuAuS00rK-pebTMAOxW41W8)
+:::note Explainer video on nominator account actions using the PolkadotJS UI
```
Comment (profradha) [inline docs/maintain/kusama/maintain-guides-how-to-nominate-kusama.md]: This is a general walkthrough video of the UI
```suggestion
:::note 
```

File: docs/maintain/kusama/maintain-guides-society-kusama.md [hunk 4]
```diff
@@ -167,8 +173,11 @@ search for "Existing Members (Challenges)". Besides that, you can earn extra KSM
 apply for the membership and requesting a tip. This is useful when a user does not have enough
 balance to reserve a deposit. The tip will be given when a user successfully joins the society.
 
-> Note: Each member can only vouch for one user at a time. A member is not required to reserve the
-> deposit when vouching for a user.
+:::info Each member can only vouch for one user at a time.
```
Comment (profradha) [inline docs/maintain/kusama/maintain-guides-society-kusama.md]:
```suggestion
:::info 
```

File: docs/maintain/kusama/maintain-guides-society-kusama.md [hunk 5]
```diff
@@ -167,8 +173,11 @@ search for "Existing Members (Challenges)". Besides that, you can earn extra KSM
 apply for the membership and requesting a tip. This is useful when a user does not have enough
 balance to reserve a deposit. The tip will be given when a user successfully joins the society.
 
-> Note: Each member can only vouch for one user at a time. A member is not required to reserve the
-> deposit when vouching for a user.
+:::info Each member can only vouch for one user at a time. 
+
+A member is not required to reserve the deposit when vouching for a user.
```
Comment (profradha) [inline docs/maintain/kusama/maintain-guides-society-kusama.md]:
```suggestion
Each member can only vouch for one user at a time. A member is not required to reserve the deposit when vouching for a user.
```

**Review by DrW3RK — CHANGES_REQUESTED** [review]

**Comment by salmad3** [comment]
@DrW3RK In these cases, the titles work well, as they are the main point to get across, where the content inside the admonition expands on the point. I'd also favour being descriptive with the titles when possible

**Comment by DrW3RK** [comment]
@DannyS03 I'm for descriptive titles in the admonitions.

The title "The following information applies to the Kusama network." - does this mean the content in the admonition applies for Kusama network or the content following the admonition applies to the Kusama network? We could just remove that title and just have this admonition that says for polkadot go to so and so, because they are indeed on a Kusama page

For this title "Explainer video on nominator account actions using the PolkadotJS UI" - this is misleading as the video is a generic explainer on the UI. Users clicking on the video link would not be listening to the nominator account actions right away

For this title "A member is not required to reserve the deposit when vouching for a user." , I felt you wanted to highlight this point to the visitor of the Wiki and the content inside the admonition is not descriptive of the title, but rather a continuation of the title's content. The next point can go in the title as well or the entire content can be in the info. I'm not very particular about this one though. Can stay the way you had it originally

So, if title is to summarize the content inside an admoniton, they do not achieve that objective. If they are to highlight a specific point, then yes. Over the review across multiple PRs, I observed this and let the titles stay. Let me know your thoughts.

**Comment by salmad3** [comment]
> @DannyS03 I'm for descriptive titles in the admonitions.
> 
> The title "The following information applies to the Kusama network." - does this mean the content in the admonition applies for Kusama network or the content following the admonition applies to the Kusama network? We could just remove that title and just have this admonition that says for polkadot go to so and so, because they are indeed on a Kusama page
> 
> For this title "Explainer video on nominator account actions using the PolkadotJS UI" - this is misleading as the video is a generic explainer on the UI. Users clicking on the video link would not be listening to the nominator account actions right away
> 
> For this title "A member is not required to reserve the deposit when vouching for a user." , I felt you wanted to highlight this point to the visitor of the Wiki and the content inside the admonition is not descriptive of the title, but rather a continuation of the title's content. The next point can go in the title as well or the entire content can be in the info. I'm not very particular about this one though. Can stay the way you had it originally
> 
> So, if title is to summarize the content inside an admoniton, they do not achieve that objective. If they are to highlight a specific point, then yes. Over the review across multiple PRs, I observed this and let the titles stay. Let me know your thoughts.

Right, I can see that - For the first two titles you mention, I had updated them to make them more clear/non misleading as you mentioned. However, we can also keep things cleaner with your suggestions. I will add them.

File: docs/maintain/kusama/maintain-guides-society-kusama.md [hunk 6]
```diff
@@ -167,8 +173,11 @@ search for "Existing Members (Challenges)". Besides that, you can earn extra KSM
 apply for the membership and requesting a tip. This is useful when a user does not have enough
 balance to reserve a deposit. The tip will be given when a user successfully joins the society.
 
-> Note: Each member can only vouch for one user at a time. A member is not required to reserve the
-> deposit when vouching for a user.
+:::info Each member can only vouch for one user at a time
```
Comment (salmad3) [inline docs/maintain/kusama/maintain-guides-society-kusama.md]:
```suggestion
:::info
```

File: docs/maintain/kusama/maintain-guides-how-to-nominate-kusama.md [hunk 7]
```diff
@@ -6,8 +6,12 @@ description: Steps on how to nominate on Kusama.
 slug: ../../maintain-guides-how-to-nominate-kusama
 ---
 
-> The following information applies to the Kusama network. If you want to nominate on Polkadot,
-> check out the [Polkadot guide](../maintain-guides-how-to-nominate-polkadot.md) instead.
+:::info This guide applies to the Kusama network
```
Comment (salmad3) [inline docs/maintain/kusama/maintain-guides-how-to-nominate-kusama.md]:
```suggestion
:::info
```

**Review by DrW3RK — APPROVED** [review]

**Pull request merged**